MAINTAINERS.md

```diff
@@ -1,3 +1,11 @@
-# MAINTAINERS
+Trestle demos was designed and open sourced by a team based at [IBM Research](https://www.research.ibm.com/) and others around the world.  The list includes:
 
-Christopher Ferris - chrisfer@us.ibm.com
+Alejandro Jose Leiva Palomo [AleJo2995](https://github.com/AleJo2995)
```

Comment:
Is this maintainers list meant to match `compliance-trestle` as this is an extension of that project or is this a meant to be custom list?

Reply:
File is now identical to that of compliance-trestle.